scripts/content-types/content-types.json

```diff
@@ -0,0 +1,358 @@
+          {"value": "Horror", "label": "Horror"},
+          {"value": "Drama", "label": "Drama"},
+          {"value": "Romance", "label": "Romance"},
+          {"value": "Western", "label": "Western"},
+          {"value": "Comedy", "label": "Comedy"},
+          {"value": "Science fiction", "label": "Science fiction"},
+          {"value": "Adventure", "label": "Adventure"},
+          {"value": "Music", "label": "Music"},
+          {"value": "Animation", "label": "Animation"},
+          {"value": "Crime film", "label": "Crime film"},
+          {"value": "History", "label": "History"},
+          {"value": "Musical genre", "label": "Musical genre"},
+          {"value": "Narrative", "label": "Narrative"},
+          {"value": "Documentary", "label": "Documentary"},
+          {"value": "Mystery", "label": "Mystery"},
+          {"value": "Noir", "label": "Noir"},
+          {"value": "Fantasy", "label": "Fantasy"},
+          {"value": "Romantic comedy", "label": "Romantic comedy"},
+          {"value": "Musical", "label": "Musical"},
+          {"value": "War", "label": "War"},
+          {"value": "Television", "label": "Television"},
+          {"value": "Fiction", "label": "Fiction"},
+          {"value": "Historical drama", "label": "Historical drama"},
+          {"value": "Sports", "label": "Sports"},
+          {"value": "Epic", "label": "Epic"},
+          {"value": "Thriller", "label": "Thriller"},
+          {"value": "Disaster", "label": "Disaster"},
+          {"value": "Martial Arts", "label": "Martial Arts"},
+          {"value": "Hindi cinema", "label": "Hindi cinema"},
+          {"value": "Satire", "label": "Satire"},
+          {"value": "Experimental", "label": "Experimental"},
+          {"value": "Slasher", "label": "Slasher"},
+          {"value": "Short", "label": "Short"},
+          {"value": "Biographical", "label": "Biographical"},
+          {"value": "Animated film", "label": "Animated film"},
+          {"value": "Narrative", "label": "Narrative"},
+          {"value": "Educational", "label": "Educational"},
+          {"value": "Cult film", "label": "Cult film"},
+          {"value": "Action/Adventure", "label": "Action/Adventure"}
+        ]
+      }
+    },
+    "rating": {
+      "param": "rating",
+      "label": "Rating",
+      "details": {
+        "field_type": "select",
+        "placeholder": "Select the content rating",
+        "options": [
+          {
+            "value": "G",
+            "label": "G"
+          },
+          {
+            "value": "PG",
+            "label": "PG"
+          },
+          {
+            "value": "PG-13",
+            "label": "PG-13"
+          },
+          {
+            "value": "R",
+            "label": "R"
+          }
+        ]
+      }
+    },
+    "trailer_id": {
+      "param": "trailerId",
+      "label": "Trailer",
+      "details": {
+        "field_type": "media_select",
+        "placeholder": "Select a trailer"
```

Comment:
Not sure what you mean by 'validation rules'. I wanted to err on the side of less schema validation in the script, just to avoid duplicating checks that the API already will do. Data should just pass through as much as possible so we don't risk accidentally mutating it in an unexpected way or having extra script-side validation that gets out of sync with the API validation.

Reply:
I only mean some validation for the fields we have, like `required`, `minLength` / `maxLength`

Reply:
Oh I see. Were there any fields you saw that needed constraints?

Reply:
I would say that all `VCH.XXX` props and `scheduleUrl` are required ones. I would also mark `playlist` for `hub` as a required prop because it just won't work without a playlist. Maybe also `markdown` ?

Reply:
Markdown and playlist are good ones. I'll mark those. I think we should leave the VCH properties alone, because it would be a weird scenario where those were missing and required, since they are auto-populated.